### PR TITLE
update dependencies to add ansi highlighting support

### PR DESCRIPTION
**Pull request by blake-mealey**
Upgrades shiki to the latest version and updates the theme and default CSS variables to support ANSI highlighting.

Note: I was having pnpm version issues locally, so I've not included a change to the pnpm-lock file.

Before:

image

After:

image

### CODE REVIEW

**Review by B2o5T — COMMENTED** [review]
can you add a screenshot before/after?

**Comment by blake-mealey** [comment]
@B2o5T sure, I've updated the description

File: .changeset/moody-maps-lay.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'nextra-theme-blog': patch
+'nextra-theme-docs': patch
+'nextra': patch
```
Comment (B2o5T) [inline .changeset/moody-maps-lay.md:4]:
```suggestion
'nextra-theme-blog': minor
'nextra-theme-docs': minor
'nextra': minor
```

**Review by B2o5T — APPROVED** [review]
thanks!

**Pull request merged**